assets/js/helpers.js

```diff
@@ -33,3 +33,16 @@ export function findSidebarCategory (items, query) {
 export function getLocationHash () {
   return window.location.hash.replace(/^#/, '')
 }
+
+export function throttle (callback, limit) {
```

Comment:
Thanks, @dignifiedquire!
I thought about it but for some reason didn't check whether we include the whole `lodash` or just some of its parts and decided to reinvent a wheel. I've updated the PR now.